collector/netstat.go

```diff
@@ -35,10 +36,19 @@ func NewNetStatCollector() (Collector, error) {
 }
 
 func (c *netStatCollector) Update(ch chan<- prometheus.Metric) (err error) {
-	netStats, err := getNetStats()
+	netStats, err := getNetStats(procNetStat)
 	if err != nil {
 		return fmt.Errorf("couldn't get netstats: %s", err)
 	}
+	SNMPStats, err := getNetStats(procSNMPStat)
```

Comment:
This'd be `snmpStats` in common Go style (local variables are never capitalized at the beginning).